eks-distro-base/Makefile

```diff
@@ -10,7 +10,7 @@ IMAGE_TAG?=$(shell date "+%F-%s")
 IMAGE?=$(IMAGE_REPO)/$(IMAGE_NAME):$(IMAGE_TAG)
 
 ifeq ($(DEVELOPMENT),true)
-	BASE_IMAGE=amazonlinux:2
+	# BASE_IMAGE=amazonlinux:2
```

Comment:
Why is this commented out?

Reply:
Yeh, sorry, I forgot to publish my comments. This is an unrelated change because I was rate limited by docker I think.